examples/machine-learning/a3-ultragpu-8g/a3ultra-slurm-blueprint.yaml

```diff
@@ -24,7 +24,7 @@ vars:
   # Image settings
   base_image:
     project: ubuntu-os-accelerator-images
-    image: ubuntu-accelerator-2204-amd64-with-nvidia-570-v20250712
+    image: ubuntu-accelerator-2204-amd64-with-nvidia-570-v20250722
```

Comment:
I would prefer to fix for 1032 package then roll forward as we have more testing around the 1032 compatibly.